docs/examples/tutorial_wavelet_api.py

```diff
@@ -0,0 +1,395 @@
+axd["signal"].legend()
+axd["phase"].plot(t, slow_oscillation_phase, alpha=0.5)
+axd["phase"].set_ylabel("Phase (rad)")
+axd["signal"].set_ylabel("Signal")
+axd["phase"].set_xlabel("Time (s)")
+[
+    axd[f].spines[sp].set_visible(False)
+    for sp in ["right", "top"]
+    for f in ["phase", "signal"]
+]
+axd["signal"].get_xaxis().set_visible(False)
+axd["signal"].spines["bottom"].set_visible(False)
+[axd[k].margins(0) for k in ["signal", "phase"]]
+axd["signal"].set_ylim(-2.5, 2.5)
+axd["phase"].set_ylim(-np.pi, np.pi)
+plt.show()
+
+# %%
+# ***
+# Adding in the 15Hz Oscillation
+# ------------------
+# Let's see what happens if we also add the 15 Hz component of the wavelet decomposition to the reconstruction. We
+# will extract the 15 Hz components, and also the 15Hz wavelet power over time. The wavelet power tells us to what
+# extent the 15 Hz frequency is present in our signal at different times.
+#
+# Finally, we will add this 15 Hz reconstruction to the one shown above, to see if it improves out reconstructed
+# signal.
+
+# Get the index of the 15 Hz frequency
+fifteen_hz_freq_idx = np.where(freqs == 15.0)[0]
+# The 15 Hz component is the real component of the wavelet decomposition at this index
+fifteenHz_oscillation = mwt[:, fifteen_hz_freq_idx].values.real
+# The 15 Hz poser is the absolute value of the wavelet decomposition at this index
+fifteenHz_oscillation_power = np.abs(mwt[:, fifteen_hz_freq_idx].values)
+
+# %%
+# Lets plot it.
+fig, ax = plt.subplots(2, constrained_layout=True, figsize=(10, 6))
+ax[0].plot(t, fifteenHz_oscillation, label="15Hz Reconstruction")
+ax[0].plot(t, fifteenHz_oscillation_power, label="15Hz Power")
+ax[1].plot(sig, label="Raw Signal", alpha=0.5)
+ax[1].plot(
+    t, slow_oscillation + fifteenHz_oscillation, label="2Hz + 15Hz Reconstruction"
+)
+[ax[i].set_ylim(-2.5, 2.5) for i in range(2)]
+[ax[i].margins(0) for i in range(2)]
+[ax[i].legend() for i in range(2)]
+[ax[i].spines[sp].set_visible(False) for sp in ["right", "top"] for i in range(2)]
+ax[0].get_xaxis().set_visible(False)
+ax[0].spines["bottom"].set_visible(False)
+ax[1].set_xlabel("Time (s)")
+[ax[i].set_ylabel("Signal") for i in range(2)]
+plt.show()
+
+# %%
+# ***
+# Adding ALL the Oscillations!
+# ------------------
+# Let's now add together the real components of all frequency bands to recreate a version of the original signal.
+
+combined_oscillations = mwt.values.real.sum(axis=1)
+
+# %%
+# Lets plot it.
+fig, ax = plt.subplots(1, constrained_layout=True, figsize=(10, 4))
+ax.plot(sig, alpha=0.5, label="Signal")
+ax.plot(t, combined_oscillations, label="Wavelet Reconstruction", alpha=0.5)
+[ax.spines[sp].set_visible(False) for sp in ["right", "top"]]
+ax.set_xlabel("Time (s)")
+ax.set_ylabel("Signal")
+ax.set_title("Wavelet Reconstruction of Signal")
+ax.set_ylim(-6, 6)
+ax.margins(0)
+ax.legend()
+plt.show()
+
+
+# %%
+# ***
+# Parametrization
+# ------------------
```

Comment:
Looking at the plot, I can't see the difference between n_cycle=1, scaling = 1 vs n_cycle=3,scaling=2.0.

Either the parameters needs to be more extreme to actually make sense of the difference, otherwise it's not the right pair of parameters

Reply:
This was a problem with the scaling of the wavelets - fixed in newest commit